### ISSUE TITLE
Demos

### ISSUE THREAD

**Issue by carloshpds** [jonrimmer/ag-grid-themes#3]
It would be nice to see some online demos 😄

**Comment by jonrimmer** [comment]
Yep. I really need to give some this whole project some attention: Get it up to date with the latest agGrid, publish to npm, provides some docs, etc.

Fortunately the changes to GitHub pages should make it a lot easier to get some demos done.

**Comment by carloshpds** [comment]
Let's do it @jonrimmer, i can help! :)
